### DISCUSSION TITLE
RAC Table colspan

### DISCUSSION THREAD

**Discussion by brenotx** [adobe/react-spectrum#7059] [Q&A] [answered]
Hi, I wonder if there is a way to use `colSpan` in the RAC `Column` component. Something like:

```tsx
<Column colSpan={2} />
```

I'd expect the `Column` component to accept all `HTMLTableCellElement` props, including `colSpan`.

<img width="711" alt="Screenshot 2024-09-21 at 9 52 12 PM" src="https://github.com/user-attachments/assets/ca3884ef-f53c-44c7-ab78-b8cf2674e89a">

Edit:
I noticed that `react-aria` `useTable` page has [nested columns](https://react-spectrum.adobe.com/react-aria/useTable.html#nested-columns) example, which is precisely what I want to do. It looks like the `Column` component coming from `react-stately` can be used to group columns, but the RAC Column can't.

Thank you in advance.

**Comment by snowystinger** [comment, marked as answer]
Thanks for the discussion.
We don't support this yet, we ran into some API issues with it in RAC and haven't had a chance to prioritize it.
https://github.com/adobe/react-spectrum/pull/5537
